documentation/ide-vscode-setup.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Openfire Source Code in Visual Studio Code IDE</title>
+    <style>
+        body {
+            background-color: #d3d6d9;
+        }
+        header {
+            background-color: #e7eaee;
+            border: 1px solid #cccccc;
+            margin-top: 10px;
+            padding-left: 20px;
+            border-bottom: 20px solid #335588;
+        }
+        header h1 {
+            font-size: 1.7em;
+            font-weight: bold;
+            color: #670e15;
+        }
+        header p {
+            color: black;
+        }
+        article {
+            background-color: #fff;
+            border: 1px solid #999;
+            padding: 40px;
+            margin: 30px;
+        }
+        img {
+            height: 250px;
+        }
+    </style>
+</head>
+<body>
+    <article>
+        <header>
+            <h1>Openfire Source Code in Visual Studio Code IDE</h1>
+            <p>
+                This is a short tutorial on how to load the Openfire source code in the Visual Studio Code IDE.
+            </p>
+            
+        </header>
+        <p>
+            This guide provides detailed information for developers that wish to
+            make changes,compiles and run the Openfire source code by using Visual Studio Code IDE.
+        </p>
+        <p>
+            For additional developer resources, please visit:
+            <a href="http://www.igniterealtime.org">http://www.igniterealtime.org</a>. 
+            The Openfire build process is based on Apache Maven. Visit the
+            <a href="https://maven.apache.org/">Maven website</a>
+            for more information.
+        </p>
+        <p>
+            Visual Studio Code or VS Code is a fast editor and ships with great editing features. It includes support for debugging,
+            embedded Git control, syntax highlighting, intelligent code completion, snippets, and code refactoring. 
+            In part thanks to the huge ecosystem of extensions that enhance and enrich the experience in VS Code, 
+            turning it into a super smart text editor, fast and lightweight. This guide focuses on the bare minimum that 
+            is needed to get started Openfire source code with VS Code.
+        </p>
+
+        <h2>Prepare your environment</h2>
+        <p>
+            The rest of this guide will assume you have at least those parts installed:
+            <ul>
+                <li><b>JAVA Development Kit (version 8)</b> </li>
+                <ul>
+                    Java 8 (JDK 1.8) must be installed and setup on your machine. To test the installation,
+                    open a shell in a Unix or a MS-DOS prompt in Windows. Check your version of
+                    Java with "java -version" -- it must version 1.8 or greater.
```

Comment:
should be "must be"
Also, currently Openfire only works with 1.8. Although maybe you can still use higher version to compile it and recently there was some changes to make it work with Java 11, but it is not released yet and not final. So, i'm not sure about this part.

Reply:
Add "be" after "it must".
Guus said 1.8 and higher is ok.